src/player.cpp

```diff
@@ -4099,10 +4099,10 @@ void PowerUps(const int A)
                             NPC[numNPCs].Type = 292;
                             NPC[numNPCs].Special5 = A;
                             NPC[numNPCs].Special6 = p.Direction;
-                            PlaySound(SFX_Throw);
+                            PlaySound(SFX_PlayerHammer);
                         }
                         else
-                            PlaySound(playerHammerSFX);
+                            PlaySound(SFX_PlayerHammer);
```

Comment:
Dude, you shouldn't modify this. You better take use the "player hammer" field and set the global flag to make the player hammer sound work. The `sounds.ini` file is already can configure to use its own player hammer sound. This was the ID holder that specifies which sound ID is used for a player hammer. Please take a look at the `sounds.ini` and enable your own player hammer sound flag rather than modifying this. It breaks a lot of compatibilities.